recipes/botan/all/conanfile.py

```diff
@@ -117,84 +124,155 @@ def config_options(self):
 
         # --single-amalgamation option is no longer available
         # See also https://github.com/randombit/botan/pull/2246
-        if tools.Version(self.version) >= '2.14.0':
+        if Version(self.version) >= '2.14.0':
             del self.options.single_amalgamation
 
+        # Support for the OpenSSL provider was removed in 2.19.2
+        if Version(self.version) >= '2.19.2':
+            del self.options.with_openssl
+
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
 
     def requirements(self):
         if self.options.with_bzip2:
             self.requires("bzip2/1.0.8")
-        if self.options.with_openssl:
-            self.requires("openssl/1.1.1o")
+        if self.options.get_safe('with_openssl', False):
+            self.requires("openssl/1.1.1s")
         if self.options.with_zlib:
-            self.requires("zlib/1.2.12")
+            self.requires("zlib/1.2.13")
         if self.options.with_sqlite3:
             self.requires("sqlite3/3.38.5")
         if self.options.with_boost:
-            self.requires("boost/1.79.0")
+            self.requires("boost/1.82.0")
```

Comment:
```suggestion
            self.requires("boost/1.83.0")
```
The latest version available in Conan Center is 1.83.0. https://conan.io/center/recipes/boost